### ISSUE TITLE
Error in errors table declaration

### ISSUE THREAD

**Issue by Slach** [yandexmobile/appmetrica-logsapi-loader#1]
Maybe use "error_id" otherwise "event_name" here?
https://github.com/yandexmobile/appmetrica-logsapi-loader/blob/master/fields/declaration.py#L195

on documentation errors table not contains "event_name" field
https://tech.yandex.ru/appmetrica/doc/mobile-api/logs/endpoints-docpage/#errors

**Comment by bamx23** [comment]
Yes, thank you. I'll fix it later. There are no posibilities to migrate DB now.
